app/models/event.rb

```diff
@@ -204,7 +204,7 @@ def progress_status
   # * +String+ -> Progress in Percent
   def calculate_progress
     result = self.progress_status
-    (100 * result.values.count(true) / result.values.count).to_s
+    (100 * result.values.count(true) / result.values.compact.count).to_s
```

Comment:
Consider a scenario where a conf has no tracks and submitter has performed all the tasks presented in the tooltip.In this case, the progress bar should be at 100%.
So when calculating progress percentage, "Add a track" task should be ignored if no track is available for the conference.